client/components/tinymce/plugins/wpcom-help/help-modal.jsx

```diff
@@ -105,26 +173,31 @@ class HelpModal extends React.Component {
 	};
 
 	render() {
+		const translate = this.props.translate;
 		const defaultText = this.props.macosx
-			? this.props.translate( 'Default shortcuts, Command + key:', { context: 'Mac shortcuts' } )
-			: this.props.translate( 'Default shortcuts, Ctrl + key:', { context: 'Windows shortcuts' } );
+			? translate( 'Default shortcuts, Command + key:', { context: 'Mac shortcuts' } )
+			: translate( 'Default shortcuts, Ctrl + key:', { context: 'Windows shortcuts' } );
 
 		const additionalText = this.props.macosx
-			? this.props.translate( 'Additional shortcuts, Control + Option + key:', {
+			? translate( 'Additional shortcuts, Control + Option + key:', {
 					context: 'Mac shortcuts',
 				} )
-			: this.props.translate( 'Additional shortcuts, Shift + Alt + key:', {
+			: translate( 'Additional shortcuts, Shift + Alt + key:', {
 					context: 'Windows shortcuts',
 				} );
 
+		const spaceFormatText = translate( 'Formatting shortcuts, key, then space:' );
+
+		const enterFormatText = translate( 'Formatting shortcuts, key, then Enter:' );
```

Comment:
Given that both `spaceFormatText` and `enterFormatText` are static, I think inlining them may be preferable. That would show them in their direct context and avoid the (albeit slight) indirection.